packages/ui-components/src/components/kv-breadcrumb-list/kv-breadcrumb-list.tsx

```diff
@@ -0,0 +1,34 @@
+import { Component, Host, h, Prop, Element } from '@stencil/core';
+import { isEmpty } from 'lodash-es';
+
+@Component({
+	tag: 'kv-breadcrumb-list',
+	styleUrl: 'kv-breadcrumb-list.scss',
+	shadow: true,
+})
+export class KvBreadcrumbList {
+	/** (optional) The separator to use between breadcrumb items */
+	@Prop() separator?: string;
+
+	/** The Host's element reference */
+	@Element() el: HTMLKvBreadcrumbListElement;
+
+	private breadcrumbItems: HTMLKvBreadcrumbItemElement[];
+
+	componentDidRender() {
+		if (!isEmpty(this.separator)) {
```

Comment:
Why we need lodash to validate if separator is null? or ''?
Other thing is, why separator don't use a default value like '/' ?

Reply:
hi miguel, the linter has a rule enforcing that only boolean types can be evaluated in if statements.

Reply:
and if we use a default value, we need this?

Reply:
we don't no, I guess I could assign a default value :P